### ISSUE TITLE
Datagrid filter closing after input 

### ISSUE THREAD

**Issue by fahrulalwan** [vmware/clarity#3348]
## Describe the bug

When i try to filter the datagrid(server-driven) somehow it works, and when it got null value, it closes the filter.

## How to reproduce

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/45678350/57426515-ab157500-7249-11e9-9c98-478989145d67.gif)

and when it comes to more than 1 filter, it produces like this.

![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/45678350/57426738-b4531180-724a-11e9-9339-25b637a6e4c6.gif)

## Expected behavior

I want the filter to not closes until the user closes themselves.

## Versions

**App**

* Angular: 7.2.15
* Node: 10.15.3
* Clarity: 1.1.3

**Device:**

* Type: Lenovo ThinkPad & Lenovo Yoga
* OS: Windows
* Only Chrome
* Version 74.0.3729.131

## Additional notes

fyi: recently i added the css for datagrid (styles.scss):
``` css
clr-datagrid {
  max-height: fit-content;
}
.datagrid {
  max-height: fit-content;
}
```

and override column width for several column:

```html
<clr-dg-column [style.width.px]="150">
  <ng-container *clrDgHideableColumn="{hidden: false}">
      Pemenang
   </ng-container>
</clr-dg-column>
```
but, the affected column doesnt use `[style.width.px]="150"`

**Comment by hippee-lee** [comment]
In the issue template we ask for a minimal reproduction. There are starter stackblitzes with Clarity already integrated here: https://stackblitz.com/@clr-team. Please reproduce the described issue so we can look at the code.

**Comment by fahrulalwan** [comment]
do you have any example API that supports server-driven datagrid?

**Comment by hippee-lee** [comment]
Every API is different and the datagrid can wirk with any API. Feel free to mock the backend. Whats important is that we see how the code for the datatgrid is implemented and can evaluate the issue from that isolation. 

Here is a [server driven stackblitz](https://stackblitz.com/edit/so-55130715-debounce-refresh-events) that mocks a back end. Feel free to fork it and simplify for your reproduction. Also, it seems like you haven't seen the documentation on server driven datagrids? https://clarity.design/documentation/datagrid/server-driven

**Comment by gnomeontherun** [comment]
Closing as stale, and likely fixed from addressing #2923.
